scripting.md

```diff
@@ -419,6 +419,25 @@ wichever other policy.
 
 See [Continuous Integration](testing.html#continuous-integration).
 
+# Bootstrapping an app
```

Comment:
Is bootstrapping and "an app" the right word ? By "bootstrapping an app" I think "cl-project" or "quickproject".

Reply:
Yeah I wasn't sure about the name, I'm not sure what to call this. Initializing maybe?